src/Apps/Order/Components/TransactionSummary.tsx

```diff
@@ -94,13 +94,7 @@ export class TransactionSummary extends React.Component<
         <StackableBorderBox flexDirection="column">
           {isOfferFlow ? (
             <>
-              <Entry
-                label="Your offer"
-                value={
-                  offerOverride ||
-                  /* TODO: || order.offer.value or whatever */ "—"
-                }
-              />
+              <Entry label="Your offer" value={offerTotal} />
```

Comment:
this should be value={offerOverride || offerTotal} so that the value updates when you type on the initial offer screen. Although maybe we should rename offerOverride to offerTotalOverride now.